### ISSUE TITLE
Thread and User relations on Message class load as null

### ISSUE THREAD

**Issue by vaishnavmhetre** [cmgmyr/laravel-messenger#307]
![screenshot from 2018-10-18 11-30-16](https://user-images.githubusercontent.com/14543290/47135476-06900d80-d2ce-11e8-9951-5eeaa1fa1799.png)

See, the user attribute is null! Same is the condition with thread relation

**Comment by cmgmyr** [comment]
Are you extending the package's `Message` class? Have you updated the config file to point to your extended `Message` class?

**Comment by vaishnavmhetre** [comment]
Well the issue resolved. I have tried using UUID as id and setting 
```
model->incrementing = false
```
in a Helper trait which wasn't actually getting executed properly in the trait. And little confusion took place in Seeders.

Little refactoring is done. Works like charm for now.

**Comment by vaishnavmhetre** [comment]
Well the confusion with the seeder was I had to create messages for _n_ threads. Well but I had to write a little lengthy chained query just to get actual user ids of the participants in the thread. 

Having a helper function in thread to directly access user object through participants would be more useful. 

Like **_hasManyThrough_**   or similar thing instead of Participant as individual model. 

We can let Paticipant stay as individual model, but then provide some optimized function directly in _Thread_  to fetch user objects of particpants.

**Comment by vaishnavmhetre** [comment]
Well the confusion with the seeder was I had to create messages for _n_ threads. But I had to write a little lengthy chained query just to get actual user object/ids of the participants in the thread. 

Having a helper function in thread to directly access user (of User::class) object through participants would be more useful. 

Like **_hasManyThrough_**   or similar thing instead of Participant as individual model. 

If necessary, we can let Paticipant stay as individual model, but then provide some optimized function directly in _Thread_  to fetch user objects through particpants table/model.
